Sources/Perception/Macros.swift

```diff
@@ -12,7 +12,7 @@
 #if canImport(Observation)
   import Observation
 
-  @available(iOS, deprecated: 17, renamed: "Observable")
+  @available(iOS, deprecated: 17.0.1, renamed: "Observable")
   @available(macOS, deprecated: 14, renamed: "Observable")
   @available(tvOS, deprecated: 17, renamed: "Observable")
   @available(watchOS, deprecated: 10, renamed: "Observable")
```

Comment:
I think we'll want the same fix for other platforms. Wanna apply the following?

```suggestion
  @available(macOS, deprecated: 14.0.1, renamed: "Observable")
  @available(tvOS, deprecated: 17.0.1, renamed: "Observable")
  @available(watchOS, deprecated: 10.0.1, renamed: "Observable")
```

Reply:
You'd want to check whether the macOS one is necessary, as macOS 14 came out after iOS 17.0.1 (technically), so the bug may not exist there.

Reply:
I raised the tvOS and watchOS as you described, I have not modified macOS - please let me know if you would prefer that @stephencelis